### ISSUE TITLE
Support array of strings

### ISSUE THREAD

**Issue by natostanco** [Stichoza/google-translate-php#26]
In the url parameters you can specify multiple `text` strings (or `q` it's the same) so that instead of splitting a long string into substrings and call each time a translation, call it only once with the array of substrings.

**Comment by natostanco** [comment]
So in order to support arrays...
`getResponse`

        if (!is_string($string) || !is_array($string)) {
            throw new InvalidArgumentException("Invalid argument provided");
        }
In this function should be specified also the guzzle aggregator (duplicateAggregator), or maybe in the options passed to the translate class? Right now I am hard coding it into guzzle because I did not find  the correct method to edit it from the http client.

then just a couple of functions, the one to detect the languagues and the one to get the correct array format...
`instanceTranslate`
```
   $array_flag = is_array($string);
   if($array_flag){
            foreach ($responseArray[0] as $itemArray) {
              foreach($itemArray as $item){
                 if (is_string($item)) {
                    $detectedLanguages[] = $item;
                }
            }
        }
        }else{
            foreach ($responseArray as $item) {
                if (is_string($item)) {
                  $detectedLanguages[] = $item;
                }
            }
        }
        if($array_flag){
            $carry = array();
            foreach($responseArray[0] as $item ){
                $carry[] = $item[0][0][0];
            }
            return $carry;
        }else{
        return array_reduce($responseArray[0], function($carry, $item) {
            $carry .= $item[0];
            return $carry;
        });
        }
```
that's it

**Comment by Stichoza** [comment]
fixed in #27
